README.md

```diff
@@ -56,7 +107,25 @@ klepto steal \
 --read-max-conns=8
 ```
 
-## Relationships
+Behind the scenes Klepto will establishes the connection with the source and target databases with the given parameters passed, and will dump 4 the tables.
```

Comment:
Is this description correct for the example file?

Reply:
Hm I just realised that for this particular example we don't need to set
```
--concurrency
--read-max-conns
--read-max-idle-conns
```

Given this config only dumps a single table (users). We could move them to the recommend section below I think